orangecontrib/spectroscopy/preprocess/__init__.py

```diff
@@ -28,6 +28,91 @@
     linear_baseline
 
 
+class MNFDenoising(Preprocess):
+
+    def __init__(self, components=None, random_state=0, svd_solver="randomized"):
```

Comment:
Remove random state and solver algorithms.

Reply:
done